src/test/java/lotto/domain/WinningNumbersTest.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class WinningNumbersTest {
+
+    @Test
+    void 로또_번호와_보너스_번호로_생성한다() {
+        WinningNumbers winningNumbers = new WinningNumbers(List.of(1, 2, 3, 4, 5, 6), 7);
+
+        assertThat(winningNumbers.getLotto().getNumbers()).contains(1, 2, 3, 4, 5, 6);
+        assertThat(winningNumbers.getBonusNumber().getValue()).isEqualTo(7);
```

Comment:
```suggestion
        assertThat(winningNumbers).isEqualTo(new WinningNumbers(List.of(1, 2, 3, 4, 5, 6), 7));
```
값을 꺼내 비교하기보다 위와 같이 객체 단위로 비교해 보면 어떨까?

Reply:
Winningnumbers가 lotto와 lottonumber를 모두 가지고 있어서 둘다 equals hashcode를 구현해야할것같네요